virtool/subtractions/data.py

```diff
@@ -83,33 +83,65 @@ async def find(self, find: str, short: bool, ready: bool, query: MultiDictProxy)
                 ).sort("name")
             ]
 
-        data = await paginate(
-            self._mongo.subtraction,
-            db_query,
-            query,
-            base_query={"deleted": False},
-            sort="name",
-            projection=PROJECTION,
-        )
+        data = await self._mongo.subtraction.aggregate(
+            [
+                {
+                    "$facet": {
+                        "total_count": [
+                            {"$count": "total_count"},
+                        ],
+                        "found_count": [
+                            {"$count": "found_count"},
+                        ],
+                        "documents": [
+                            {
+                                "$match": query,
```

Comment:
I believe that as written the aggregation pipeline will include documents which documents which have their `deleted` flag set to true. None of the facets should count samples which are `deleted`